{
	"id": 52531491,
	"number": 9404,
	"state": "closed",
	"title": "runtime: panic from deferred function during Goexit has misleading stack trace",
	"body": "Running this program on go1.4:\r\n\r\n```go\r\npackage main\r\n\r\nimport \"runtime\"\r\n\r\nfunc main() {\r\n\tdefer panic(\"boom\")\r\n\truntime.Goexit()\r\n}\r\n```\r\n\r\nResults in this output:\r\n\r\n```text\r\npanic: boom\r\n\r\ngoroutine 1 [running]:\r\nruntime.Goexit()\r\n\t/usr/local/go/src/runtime/panic.go:288 +0xf6\r\nmain.main()\r\n\t/home/titanous/goexit.go:7 +0x7a\r\n...\r\n```\r\n\r\nThis stack trace does not indicate the real source of the panic, and is confusing.",
	"user": {
		"login": "titanous",
		"id": 13026,
		"type": "User",
		"site_admin": false
	},
	"labels": [
		{
			"name": "FrozenDueToAge"
		}
	],
	"comments": 1,
	"closed_at": "2014-12-21T19:40:45Z",
	"created_at": "2014-12-19T21:36:45Z",
	"updated_at": "2016-06-25T01:57:00Z"
}
